_posts/2023-06-16-deploy-on-kubernetes-with-helm.adoc

```diff
@@ -0,0 +1,400 @@
+----
+
+
+## Deploy the Application on Kubernetes
+
+Once the `wildfly` Chart is added, we can install a Helm *release* by using the `helm install` command with the name of the release as well as a YAML configuration file that contains all the settings to build and deploy the application:
+
+[source]
+----
+$ helm install helloworld-app \
+    wildfly/wildfly \
+    -f charts/values.yaml
+
+NAME: helloworld-app
+LAST DEPLOYED: Wed Jun 14 18:45:38 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+To follow the deployment of your application, run:
+
+$ kubectl get deployment helloworld-app -w
+----
+
+As you can read above, the output of the `helm install` command contains instructions to follow the installation of the application.
+
+We can watch its deployment by running:
+
+[source]
+----
+$ kubectl get deployment helloworld-app -w
+NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+helloworld-app   1/1     1            1           30s
+----
+
+The application is now available and we can query it (you might need to wait some seconds to see our application up and running).
+
+In order to reach your application from the browser you could port-forward port 8080 to your host:
+
+[source]
+----
+$ kubectl port-forward $(oc get pod -l app.kubernetes.io/instance=helloworld-app -o name) 8080:8080
+Forwarding from 127.0.0.1:8080 -> 8080
+...
+----
+
+Note: If your application has a different name you can retrieve your pod name by executing:
+`$ kubectl get pod`
+
+Then in the output you will see your pod name.
+
+Using the 'pod name' execute:
+`$ kubectl port-forward your_pod_name 8080:8080`
+
+Now open your browser and browse URL 'localhost:8080', your application will respond: Hello World!
+
+Alternately you can curl it from your terminal:
+
+[source]
+----
+$ curl localhost:8080
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<!-- Plain HTML page that kicks us into the app -->
+
+<html>
+    <head>
+        <meta http-equiv="Refresh" content="0; URL=HelloWorld">
+    </head>
+</html>
+----
+
+The application is up and running.
+
+## Conclusion
+It is possible to run you application with very simple steps.
+
+All is necessary is:
```

Comment:
s/All is/All that is/g

Reply:
Also, I suggest

s/is:/is to:/g

and then drop the 'to' at the beginning of the 3 bullets.

That reads more naturally to me, but YMMV.

Reply:
Thank you, I addressed your suggestions.